lib/bootscale/entry.rb

```diff
@@ -13,6 +13,7 @@ def initialize(path)
     end
 
     def features
+      return [] unless @path.start_with?('/'.freeze)
```

Comment:
No, because order is important: 

if `$LOAD_PATH = ['lib/', '/app/']` and that both `lib/x.rb` and `/app/x.rb` exists.

When you `require x.rb` it should give you `lib/x.rb`, but with this PR it would give you `/app/x.rb`.